classes/easysplash-animation.bbclass

```diff
@@ -2,19 +2,16 @@
 # easysplash-animation.bbclass allows for easy packaging of EasySplash
 # animation packages.
 #
-# Copyright (C) 2015, 2020 O.S. Systems Softwares Ltda.  All Rights Reserved
+# Copyright (C) 2015, 2021 O.S. Systems Softwares Ltda.  All Rights Reserved
 # Released under the MIT license (see packages/COPYING)
 
-inherit update-alternatives allarch
+inherit update-alternatives
 
-ALTERNATIVE_${PN} += "bootanimation"
-ALTERNATIVE_TARGET[bootanimation] = "/lib/easysplash/${PN}.zip"
-ALTERNATIVE_LINK_NAME[bootanimation] = "/lib/easysplash/bootanimation.zip"
-ALTERNATIVE_PRIORITY[bootanimation] = "50"
+ALTERNATIVE_${PN} += "animation"
+ALTERNATIVE_TARGET[animation] = "${base_libdir}/easysplash/${PN}"
+ALTERNATIVE_LINK_NAME[animation] = "${base_libdir}/easysplash/animation"
+ALTERNATIVE_PRIORITY[animation] = "50"
 
-do_install_append() {
-    install -Dm 0644 ${S}/*.zip ${D}/lib/easysplash/${PN}.zip
-}
-
-FILES_${PN} += "/lib/easysplash/${PN}.zip"
 RDEPENDS_${PN} += "easysplash"
+
+FILES_${PN} += "${base_libdir}/easysplash/${PN}"
```

Comment:
Move the `FILES_${PN}` on top of existing `RDEPENDS_${PN}` so we reduce the diff.